### PR TITLE
Fix typos in Developer Website's Projects section on front page

### LINKED ISSUE #1570: Fix Casing in Developer Website's Projects Section on Front Page
# Fix Casing in Developer Website's Projects Section on Front Page

## 🚀 Goal
We want to ensure our website's different pages don't have any mistakes.

## 🤔 Background
This issue focuses specifically on fixing typos in the projects page of our website. Your contributions here make a difference!

## 🤝 How You Can Contribute
We've broken it down into specific typos for you to address. Every contribution counts!

### **❗You must only assign yourself one task at a time to give everyone a chance to participate.❗**
You may assign yourself your next task after your current task is reviewed & accepted.
1. /projects Page
    - [ ] Update casing from "TBDEX" to "tbDEX"
image
2. Make sure all tests are passing.

## 🌟 Resources
- [Developer Website](https://developer.tbd.website/)

## Getting Started
1. Comment ".take" on this issue to get assigned
2. Fork the repository and create a new branch for this task
3. Follow the tasks outlined above
4. Submit a pull request with your changes
6. Respond to any feedback during the review process

## Questions?
If you have any questions or need clarification, please comment on this issue or join our [Discord](https://discord.gg/tbd) community.

Happy coding! 🎉

### PR DESCRIPTION
**Pull request by Johnnyevans32**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ New Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 👷 Example Application
- [ ] 🧑‍💻 Code Snippet
- [ ] 🎨 Design
- [ ] 📖 Content
- [ ] 🧪 Tests
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

This PR fixes the letter casing of tbDEX in the "Projects" section of the Developer Website's front page.

## Related Tickets & Documents

Resolves #1570 

## Mobile & Desktop Screenshots/Recordings
Untitled 103

## Added code snippets?
- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 general documentation
- [x] 🙅 no documentation needed

### CODE REVIEW

**Review by acekyd — APPROVED** [review]

**Review by angiejones — CHANGES_REQUESTED** [review]
previewing...dont merge yet

File: site/src/components/Project.jsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const Project = ({
         <img src={icon} alt="" className="my-0" />
       </div>
       <div className="pb-6 tablet:pb-[0.875rem]">
-        <h2 className="h2-caps my-0 text-primary-yellow">{title}</h2>
```
Comment (blackgirlbytes) [inline site/src/components/Project.jsx:20]: Hiii! Can you help me understand why this line was changed?

File: site/src/css/custom.css [hunk 2]
```diff
@@ -1,9 +1,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
-
```
Comment (blackgirlbytes) [inline site/src/css/custom.css:4]: hey any chance we can revert the changes that happen to this file? I don't think we want to change css files for the entire site? Thank youuuu so much!

**Review by blackgirlbytes — CHANGES_REQUESTED** [review]

**Review by angiejones — COMMENTED** [review]
team is on it

**Review by angiejones — APPROVED** [review]

File: site/src/components/Project.jsx [hunk 3]
```diff
@@ -17,7 +17,7 @@ const Project = ({
...
+        <h2 className="text-2xl my-0 text-primary-yellow">{title}</h2>
```
Comment (blackgirlbytes) [inline site/src/components/Project.jsx:20]:
```suggestion
        <h2 className="h2-caps my-0 text-primary-yellow">{title}</h2>
```

File: site/src/css/custom.css (hunk 2 above)
Comment (Johnnyevans32) [reply site/src/css/custom.css:4]: no changes were made, just some auto format changes, I will revert changes now.

File: site/src/components/Project.jsx (hunk 1 above)
Comment (Johnnyevans32) [reply site/src/components/Project.jsx]: The CSS on `h2-caps` included an uppercase styling on the project titles causing tbDEX always to read TBDEX uppercased so I removed the class name and replaced it with the CSS class name that doesn't alter the letter casing of the project title

Comment (Johnnyevans32) [reply site/src/components/Project.jsx:20]: > The CSS on `h2-caps` included an uppercase styling on the project titles causing tbDEX always to read TBDEX uppercased so I removed the class name and replaced it with the CSS class name that doesn't alter the letter casing of the project title

.h2-caps { @apply uppercase font-heading text-2xl; }

so I replace h2-caps with just text-2xl

File: site/src/css/custom.css (hunk 2 above)
Comment (blackgirlbytes) [reply site/src/css/custom.css]: thank you!!!

File: site/src/components/Project.jsx (hunk 1 above)
Comment (Johnnyevans32) [reply site/src/components/Project.jsx:20]: done

File: site/src/content/projects/project-index/projects.js [hunk 4]
```diff
@@ -13,22 +13,22 @@ export const content = {
   projects: [
     {
       icon: '/img/article-icon.svg',
-      title: 'Web5',
+      title: 'WEB5',
```
Comment (angiejones) [inline site/src/content/projects/project-index/projects.js:16]:
```suggestion
      title: 'Web5',
```

**Review by blackgirlbytes — APPROVED** [review]

**Pull request merged**